src/scenes/Activities/Favorite/store.js

```diff
@@ -0,0 +1,92 @@
+import { observable, action, runInAction, reaction } from 'mobx';
+import _ from 'lodash';
+import { Routes, SortBy, OracleStatus } from 'constants';
+import { queryAllTopics, queryAllOracles } from '../../../network/graphql/queries';
+import Topic from '../../../stores/models/Topic';
+import Oracle from '../../../stores/models/Oracle';
+
+const INIT_VALUES_FAVPAGE = {
+  loaded: false, // loading state?
+  loadingMore: true, // for loading icon?
+  displayList: [], // data list
+};
+
+export default class {
```

Comment:
nit: add the class name. I find its easier to tell what class you are looking at.